### PR TITLE
added styles for golang syntax highlighting

### PR DESCRIPTION
**Pull request by xh3b4sd**
This PR provides more decent syntax highlighting for https://golang.org/.

### CODE REVIEW

File: styles/Languages/go.less [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// ---------------------------------------------------------------------------
+//  Languages: Go
+// ---------------------------------------------------------------------------
+
+    .source.go {
+            .entity.function,
```
Comment (thomaslindstrom) [inline styles/Languages/go.less]: Please use 4 spaces for tabs

File: styles/Languages/go.less [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            .keyword,
+            .support.function.builtin,
+            .type {
+              color: @color-blue;
+            }
+            .entity.name.type,
```
Comment (thomaslindstrom) [inline styles/Languages/go.less]: Please add a new line before new selectors

Comment (xh3b4sd) [inline styles/Languages/go.less]: Done.

**Comment by xh3b4sd** [comment]
@thomaslindstrom I applied your requested changes. Please have another look if you do not mind. Thanks for the opportunity of collaborating with you.

**Comment by thomaslindstrom** [comment]
Thanks!

**Pull request merged**